atlp-rwanda/Dominators-Phantom-BE
# resolved

What does this PR do?
Creates or updates the user's profile

Description of Task to be completed?
As operator or driver, I want to be able to update information on my profile, so that others could see updated information about me.
Appropriate error messages are displayed on unsuccessful attempts

How should this be manually tested?
git pull this branch
run npm install
check env.example file and add JWT_SECRET and JWT_EXPIRES_IN to your .env file
Do migrations with npm run migrate
run npx sequelize-cli db:seed:all to create pre-configured credentials for seeders
run npm run dev
Test these routes
POST /api/v1/users/:id/update/ in POSTMAN or any other platform for using API
